src/cdk/explorer/nodes/constructNode.ts

```diff
@@ -79,3 +79,19 @@ export class ConstructNode extends AWSTreeNodeBase {
         return entities
     }
 }
+
+/** 
+    * Determines if a CDK construct is of type state machine
+    * 
+    * @param construct CDK construct
```

Comment:
According to the [official docs](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns), we should the parameter type before the parameter name, then add a `-` to separate the parameter name from its description.